content/docs/rfcs/56/README.md

```diff
@@ -0,0 +1,412 @@
+  bytes community_id = 5;
+}
+
+message CommunityRequestToLeave {
+  // The Lamport timestamp of the request
+  uint64 clock = 1;
+  // The community public key
+  bytes community_id = 2;
+}
+
+message CommunityDescription {
+  // The Lamport timestamp of the message
+  uint64 clock = 1;
+  // A mapping of members in the community to their roles
+  map<string,CommunityMember> members = 2;
+  // The permissions of the Community
+  CommunityPermissions permissions = 3;
+  // The metadata of the Community
+  ChatIdentity identity = 5;
+  // A mapping of chats to their details
+  map<string,CommunityChat> chats = 6;
+  // A list of banned members
+  repeated string ban_list = 7;
+  // A mapping of categories to their details
+  map<string,CommunityCategory> categories = 8;
+  // The admin settings of the Community
+  CommunityAdminSettings admin_settings = 10;
+  // If the community is encrypted
+  bool encrypted = 13;
+  // The list of tags
+  repeated string tags = 14;
+}
+```
+
+Note: The usage of the clock is described in the [Clock](#clock) section.
+
+## Content topic usage
+
+"Content topic" refers to the field in [14/WAKU2-MESSAGE](/spec/14/#message-attributes), further elaborated in [10/WAKU2](/spec/10/#overview-of-protocol-interaction).
+
+### Advertising a Community
+
+The content topic that the community is advertised on MUST be derived from the public key of the community.
+The content topic MUST be the hex-encoded keccak-256 hash of the compressed (32 bytes) public key of the community.
+
+```
+contentTopic = hex(keccak256(compressedPublicKey))
+```
+
+### Community channels/chats
+
+The unique identifier for a community channel/chat is the chat id.
+<!-- Don't enforce any constraints on the unique id generation -->
+The content topic that Community channels/chats use MUST be the hex-encoded keccak-256 hash of the public key of the community concatenated with the chat id.
+
+```
+contentTopic = hex(keccak256(compressedPublicKey + chatId))
+```
+
+
+### Community event messages
+
+Requests to leave, join, kick and ban, as well as key exchange messages, MUST be sent to the content topic derived from the public key of the community.
+The content topic MUST be the hex-encoded keccak-256 hash of the public key of the community.
+
+```
+contentTopic = hex(keccak256(publicKey))
+``` 
+
+## Community Management
+
+The flows for Community management are as described below.
+
+### Community Creation Flow
+
+1. The Community owner generates a public/private key pair.
+2. The Community owner configures the Community metadata, according to the wire format "CommunityDescription".
+3. The Community owner publishes the Community metadata on a content topic derived from the public key of the Community. 
+the Community metadata SHOULD be encrypted with the public key of the Community. <!-- TODO: Verify this-->
+The Community metadata MAY be sent during fixed intervals, to ensure that the Community metadata is available to members.
```

Comment:
Would be good to define the current practice (what interval is used?) as this will help with the correctness of simulations.

Reply:
I could not find instances of the community description being published periodically in status-go except when a change in the community description or control message happens.

Reply:
https://github.com/status-im/status-go/blob/develop/protocol/messenger_communities.go#L191

Reply:
As per @felicio, https://discord.com/channels/864066763682218004/1065146990540902400/1067769073678680126
It's broadcasted every hour.